### ISSUE TITLE
file_get_contents(http://localhost:7777/c3/report/clear): failed to open stream: Address not available

### ISSUE THREAD

**Issue by OtezVikentiy** [Codeception/Codeception#5139]
#### What are you trying to achieve?
i'm trying to get the full report about codecoverage
#### What do you get instead?
file_get_contents(http://localhost:7777/c3/report/clear): failed to open stream: Address not available
> Provide console output if related. Use `-vvv` mode for more details.

```bash
bash-4.3# php vendor/bin/codecept run  --coverage --coverage-xml --coverage-html
Cannot load Xdebug - it was already loaded
Codeception PHP Testing Framework v2.4.5
Powered by PHPUnit 7.2.7 by Sebastian Bergmann and contributors.

Acceptance Tests (0) -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

In ErrorHandler.php line 83:
                                                                                                          
  file_get_contents(http://localhost:7777/c3/report/clear): failed to open stream: Address not available  
                                                                                                          

run [-o|--override OVERRIDE] [-e|--ext EXT] [--report] [--html [HTML]] [--xml [XML]] [--tap [TAP]] [--json [JSON]] [--colors] [--no-colors] [--silent] [--steps] [-d|--debug] [--coverage [COVERAGE]] [--coverage-html [COVERAGE-HTML]] [--coverage-xml [COVERAGE-XML]] [--coverage-text [COVERAGE-TEXT]] [--coverage-crap4j [COVERAGE-CRAP4J]] [--coverage-phpunit [COVERAGE-PHPUNIT]] [--no-exit] [-g|--group GROUP] [-s|--skip SKIP] [-x|--skip-group SKIP-GROUP] [--env ENV] [-f|--fail-fast] [--no-rebuild] [--] [<suite> [<test>]]


Fatal error: Uncaught RuntimeException: Command Did Not Finish Properly in /app/vendor/codeception/codeception/src/Codeception/Subscriber/ErrorHandler.php on line 101

Call Stack:
    0.0003     406128   1. {main}() /app/vendor/codeception/codeception/codecept:0
    0.1121    7216472   2. Codeception\Application->run() /app/vendor/codeception/codeception/codecept:42
    0.1122    7216472   3. Codeception\Application->run() /app/vendor/codeception/codeception/src/Codeception/Application.php:108

RuntimeException: Command Did Not Finish Properly in /app/vendor/codeception/codeception/src/Codeception/Subscriber/ErrorHandler.php on line 101

Call Stack:
    0.0003     406128   1. {main}() /app/vendor/codeception/codeception/codecept:0
    0.1121    7216472   2. Codeception\Application->run() /app/vendor/codeception/codeception/codecept:42
    0.1122    7216472   3. Codeception\Application->run() /app/vendor/codeception/codeception/src/Codeception/Application.php:108
    0.1879   10880016   4. Codeception\Subscriber\ErrorHandler->shutdownHandler() /app/vendor/codeception/codeception/src/Codeception/Subscriber/ErrorHandler.php:0

```

### Details

* Codeception version: 2.3
* PHP Version: 7.2
* Operating System: ubuntu 16.04
* Installation type: Composer

```yml
paths:
  tests: tests
  output: tests/_output
  data: tests/_data
  support: tests/_support
  envs: tests/_envs
actor_suffix: Tester
extensions:
  enabled:
    - Codeception\Extension\RunFailed
coverage:
  enabled: true
  remote: false
  include:
    - src/*
  exclude:
    - src/Entity/*
  c3_url: 'http://localhost:7777'
params:
  - .env.test
```

**Comment by jha1395** [comment]
I think you will have to install coverage driver .

**Comment by OtezVikentiy** [comment]
@jha1395 What do you mean coverage driver?
When i run `php vendor/bin/codecept run unit --coverage-html` or `php vendor/bin/codecept run functional --coverage-html` it works fine, but only for the exact type of tests and i can not see the full picture of coverage because one overwrites another. And i need to do so that coverage worked both on functional and unit at once.

**Comment by jha1395** [comment]
Please check  under  php/etc/php.ini file

> zend_extension="/Applications/mampstack-7.1.18-1/php/lib/php/extensions/xdebug.so"
> xdebug.remote_enable=true
> xdebug.remote_host=127.0.0.1
> xdebug.remote_port=9000
> xdebug.remote_handler=dbgp


these all debug should be enabled.

As you can see in my system  it is working
![screen shot 2018-08-28 at 10 57 24 am](https://user-images.githubusercontent.com/30498249/44702546-9fb25d00-aab1-11e8-98c3-7b22911cc851.png)

**Comment by Naktibalda** [comment]
` failed to open stream: Address not available` looks like networking error.
I have no idea why you get such error on client side.

**Comment by amcnea** [comment]
@OtezVikentiy I know this has been closed for a while, but I had the same issue.  If I ran unit or functional tests on their own, everything worked.  But running them together produced the same error message for me.

It turns out that the issue was the Acceptance tests (which are empty in my project).  Just by having the default Acceptance yaml file, it was loading up some module which causes the LocalServer.php to make that request and die.

I simply removed the empty default Acceptance tests files from my project and everything works now.
